src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -3209,7 +3209,7 @@ because xbuild doesn't support framework reference assemblies.
     _Sign;
   </SignAndroidPackageDependsOn>
 </PropertyGroup>
-<Target Name="SignAndroidPackage" DependsOnTargets="$(SignAndroidPackageDependsOn)">
+<Target Name="SignAndroidPackage" DependsOnTargets="$(SignAndroidPackageDependsOn)" BeforeTargets="Publish">
```

Comment:
Yeah this could use `$(PublishDependsOn)`: https://github.com/microsoft/msbuild/blob/0c507a29b6d59340ef5221f66df5d953c1049ba1/src/Tasks/Microsoft.Common.CurrentVersion.targets#L5181

Reply:
I've decided to cut `dotnet publish` considerations for this iteration.